src/containers/projects/Projects.js

```diff
@@ -19,6 +19,7 @@ export default function Projects() {
   }, []);
 
   function getRepoData() {
+    const CUSTOM_ERR_MSG = 'Because of this Error, nothing is shown in place of Projects section. Projects section not configured'
```

Comment:
I think we can change the error messages to something similar to the Twitter component, i.e `Can't load? Check privacy protection settings`

This is because the error message would be user facing and can also be caused by strict script rules on user's browser. Also, the project shouldn't build correctly if there's some error in fetching.